checkov/cdk/checks/typescript/LambdaFunctionLevelConcurrentExecutionLimit.yaml

```diff
@@ -0,0 +1,51 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_115
+  name: Ensure that AWS Lambda function is configured for function-level concurrent execution limit
+  category: GENERAL_SECURITY
+  framework: cdk
+scope:
+  languages:
+    - typescript
+definition:
+  or:
+    - pattern: new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY> })
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY>, reservedConcurrentExecutions: $NUM, <ANY> }, <ANY>)
+        - metavariable: $PACKAGE
+          comparison: $PACKAGE == "aws-cdk-lib"
+    - pattern: new $PACKAGE.aws_lambda.CfnFunction(<ANY>, <ANY>, { <ANY> })
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.CfnFunction(<ANY>, <ANY>, { <ANY>, "reservedConcurrentExecutions": $NUM, <ANY> }, <ANY>)
+        - metavariable: $PACKAGE
+          comparison: $PACKAGE == "aws-cdk-lib"
+    - pattern: new $PACKAGE.aws_sam.CfnFunction(<ANY>, <ANY>, { <ANY> })
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_sam.CfnFunction(<ANY>, <ANY>, { <ANY>, reservedConcurrentExecutions: $NUM, <ANY> }, <ANY>)
+        - metavariable: $PACKAGE
+          comparison: $PACKAGE == "aws-cdk-lib"
+    - pattern: |
+        $X = { <ANY> }
```

Comment:
same as I wrote in LambdaInVPC

Reply:
i don't have now proper way to do it as you except